packages/components/src/decorative-boxes/index.tsx

```diff
@@ -0,0 +1,201 @@
+import style from './style.module.css';
+
+interface BoxPosition {
+  top?: number;
+  left?: number;
+  bottom?: number;
+  right?: number;
+}
+
+export interface DecorativeBoxesProps
+  extends React.ComponentPropsWithoutRef<any> {
+  children: JSX.Element;
+  boxSize?: number;
+  box1Position?: BoxPosition;
+  box2Position?: BoxPosition;
+  box1Color?: string;
+  box2Color?: string;
+}
+
+/*
+Incomplete todo-list:
+TODO: Implement random default colors
+TODO: Investigate if it is easier/more consistent
+ to just pass bottom/right as styling instead of 
+ translating to top/left every time
+*/
+
+const DecorativeBoxes: React.FC<DecorativeBoxesProps> = ({
+  children,
+  boxSize = 60,
+  box1Position,
+  box2Position,
+  box1Color = 'var(--color-secondary2__tint4)',
```

Comment:
Det skal være mulig å referere til navngitte farger via theme her, men husker ikke hvordan i farten. Men kan se på dette kanskje? https://github.com/varianter/styleguide/blob/master/web/styleguide/color-grid/index.tsx

Reply:
Usikker på hva du mener, sånn som det gjøres her henter den jo secondary color 2, tint 4 (som forøvrig er #cdf8f4). Tanken var at man skulle kunne velge mellom enten helt custom farger ved å oppgi en hex-kode, eller referere til temafargene våre slik det gjøres når man bruker CSS-variablene som ligger i rooten av sida.

Er kanskje litt lite intuitivt at det skal ligge var() rundt i farge-attributten da.